### PR TITLE
[Governance] add execution status to proposals table

### PR DESCRIPTION
**Pull request by zihan-aptos**
* make execution status helper functions and reuse them in the proposal header and proposals table
* add execution status to proposals table
Screen Shot 2022-08-18 at 12 21 18 PM

### CODE REVIEW

**Comment by msavir** [comment]
Great idea! Although imo the ui looks a bit busy, what do you think on having an info icon (maybe https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Ainfo%3A) and when hover it would say "waiting to be executed"?

File: src/pages/Governance/utils.tsx [hunk 1]
```diff
@@ -122,3 +123,25 @@ export function renderStatusIcon(
       return assertNever(proposalState);
   }
 }
+
+export function getExecutionStatusColor(isResolved: boolean): string {
+  return isResolved ? primaryColor : warningColor;
+}
+
+export function renderExecutionStatusIcon(isResolved: boolean): JSX.Element {
```
Comment (zacharydenton) [inline src/pages/Governance/utils.tsx:131]: For stuff like this, just make it a component. `export function ExecutionStatusIcon({isResolved}: {isResolved: boolean}) { ... }`

**Review by zacharydenton — APPROVED** [review]
LGTM

The "Waiting to be executed" copy is kind of funny. Maybe just "Pending"

**Comment by zihan-aptos** [comment]
> 

I'd keep it as consistent as possible as the proposal page, meaning that I'd prefer show two status. Although I agree that the UI can be more clean. Two solutions I'm thinking:
1. As Zach mentioned below, the phrase "Waiting to be executed" is a bit redundant. If we can come up with a shorter but accurate phrase, the UI would look better. 
2. I'm thinking if we should just have 5 status for voting+execution instead having two set of status. They can be: Pending, Failed, Rejected, Passed (implying waiting for execution), Executed. 
It'll be a good discussion to have with the team. Let me bring this to a broader group.

**Pull request merged**